src/test/java/seedu/address/logic/parser/EditNoteCommandParserTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.EditNoteCommand;
+import seedu.address.model.startup.Note;
+
+public class EditNoteCommandParserTest {
+
+    private static final String MESSAGE_INVALID_FORMAT =
+            String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditNoteCommand.MESSAGE_USAGE);
+
+    private EditNoteCommandParser parser = new EditNoteCommandParser();
+
+    @Test
+    public void parse_validArgs_returnsEditNoteCommand() {
```

Comment:
Perhaps we can comment out the failing test cases first so that it can pass the build checks